### PR TITLE
Added tiptop as a package.

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/staged-recipes Ready for review!

File: recipes/tiptop/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tiptop-{{ version }}.tar.gz
+  sha256: cebdce70347eff7dc621dc0c9bd2167254db75ce490db7ea216cc6d970d858bc
+
+build:
+  # Cannot be noarch due to binary executables being build
```
Comment (xhochy) [inline recipes/tiptop/meta.yaml:14]: Why is then no compiler listed here?

Comment (thewchan) [reply recipes/tiptop/meta.yaml:14]: @xhochy  My mistake, I forget the entry point. I can switch back to noarch. It doesn't work with win though.

**Comment by thewchan** [comment]
@conda-forge/staged-recipes Ready for review@

File: recipes/tiptop/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipes/tiptop/meta.yaml:14]: Ah, by "binary executables" you mean entry points? In that case, yes, it should be marked as `noarch` please.

Comment (thewchan) [reply recipes/tiptop/meta.yaml:14]: It's already been changed to noarch @jaimergp @xhochy @conda-forge/staged-recipes

**Pull request merged**